core-services/release-controller/_releases/release-ocp-4.12.json

```diff
@@ -416,12 +416,6 @@
       "prowJob": {
         "name": "periodic-ci-openshift-osde2e-main-nightly-4.12-rosa-hcp"
       }
-    },
-    "fips-scan": {
-      "optional": true,
-      "prowJob": {
-        "name": "periodic-ci-openshift-release-master-nightly-4.12-fips-payload-scan"
```

Comment:
Shouldn't all these simply be a rename to: `periodic-ci-openshift-release-master-fips-payload-scan`?